devices/homeetec.js

```diff
@@ -0,0 +1,360 @@
+                await reporting.bind(device.getEndpoint(1), coordinatorEndpoint, ['genOnOff']);
+            },
+        },
+        {
+            fingerprint: [{
+                modelID: 'TS0002',
+                manufacturerName: '_TZ3000_in5qxhtt',
+            }],
+            model: 'TS0002_switch_2_gang',
+            vendor: 'TuYa',
+            description: '2-Gang switch with backlight',
+            fromZigbee: [fz.on_off, tuya.fz.power_on_behavior_2, tuya.fz.backlight_mode_off_on],
+            toZigbee: [tz.on_off, tuya.tz.power_on_behavior_2, tuya.tz.backlight_indicator_mode_2],
+            exposes: [
+                e.switch().withEndpoint('l1'),
+                e.switch().withEndpoint('l2'),
+                e.power_on_behavior(),
+                exposes.binary('backlight_mode', ea.STATE_SET, 'on', 'off').withDescription('Backlight mode'),
+            ],
+            endpoint: (device) => {
+                return {'l1': 1, 'l2': 2};
+            },
+            meta: {multiEndpoint: true, disableDefaultResponse: false},
+            configure: async (device, coordinatorEndpoint, logger) => {
+                await tuya.configureMagicPacket(device, coordinatorEndpoint, logger);
+                await reporting.bind(device.getEndpoint(1), coordinatorEndpoint, ['genOnOff']);
+                await reporting.bind(device.getEndpoint(2), coordinatorEndpoint, ['genOnOff']);
+            },
+        },
+        {
+            fingerprint: [{
+                modelID: 'TS0003',
+                manufacturerName: '_TZ3000_pv4puuxi',
+            }],
+            model: 'TS0003_switch_3_gang',
+            vendor: 'TuYa',
+            description: '3-Gang switch with backlight',
+            fromZigbee: [fz.on_off, tuya.fz.power_on_behavior_2, tuya.fz.backlight_mode_off_on],
+            toZigbee: [tz.on_off, tuya.tz.power_on_behavior_2, tuya.tz.backlight_indicator_mode_2],
+            exposes: [
+                e.switch().withEndpoint('left'),
+                e.switch().withEndpoint('center'),
+                e.switch().withEndpoint('right'),
+                e.power_on_behavior(),
+                exposes.binary('backlight_mode', ea.STATE_SET, 'on', 'off').withDescription('Backlight mode'),
+            ],
+            endpoint: (device) => {
+                return {'left': 1, 'center': 2, 'right': 3};
+            },
+            meta: {multiEndpoint: true},
+            configure: async (device, coordinatorEndpoint, logger) => {
+                await tuya.configureMagicPacket(device, coordinatorEndpoint, logger);
+                await reporting.bind(device.getEndpoint(1), coordinatorEndpoint, ['genOnOff']);
+                await reporting.bind(device.getEndpoint(2), coordinatorEndpoint, ['genOnOff']);
+                await reporting.bind(device.getEndpoint(3), coordinatorEndpoint, ['genOnOff']);
+            },
+        },
+        {
+            fingerprint: [{
```

Comment:
switches worked, backlight didn't since it was not supported in the datapoints.key object, added it temporary and it works:
```javascript
tuya.tz.datapoints.key = [...tuya.tz.datapoints.key, 'backlight_mode'];
```

meaning need to add it to the original declaration as additional key.

in addition, the convertedKeys taking for granted that when the multiEndpoint is set to `true`, that the exposed object is also associated with endpoint, updated the if inline statement to fix it:
```javascript
const convertedKey = meta.mapped.meta.multiEndpoint && (meta.endpoint_name || false) ? `${key}_${meta.endpoint_name}` : key;
```

Reply:
added that fix to `lib/tuya.js` (line 1667)